scripts/pathseq/wdl/README.md

```diff
@@ -3,43 +3,44 @@
 ### Setting up parameter json file for a run
 
 To get started, *copy* the ``pathseq_pipeline_template.json`` for the workflow and modify the parameters accordingly.
-DO NOT commit your json file to this repo. This file has reasonable default parameters.
+DO NOT commit your json file to this repo.
 
-PathSeq reference files are available in the [GATK Resource Bundle](https://software.broadinstitute.org/gatk/download/bundle) (located [here](ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/beta/PathSeq)).
+PathSeq reference files are available in the [GATK Resource Bundle](https://software.broadinstitute.org/gatk/download/bundle) at `gs://gatk-best-practices/pathseq`.
```

Comment:
That broad link is giving me a 404.  Maybe just take it out?

Reply:
Looks like they moved it to https://gatk.broadinstitute.org/hc/en-us/articles/360035890811-Resource-bundle